test/Microsoft.AspNet.Server.Kestrel.FunctionalTests/ReuseStreamsTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http.Features;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Microsoft.AspNet.Server.Kestrel.FunctionalTests
+{
+    public class ReuseStreamsTests
+    {
+        [Fact]
+        public async Task ReuseStreamsOn()
```

Comment:
I'm sorry if I'm missing something obvious here, but I don't see how this test is effectively validating that the stream are being reused.